tomcat/config.sls

```diff
@@ -28,6 +28,17 @@ tomcat tomcat_conf:
     - watch_in:
       - service: tomcat package installed and service running
 
+{% if tomcat.catalina_tmpdir|default %}
+catalina tmpdir:
+  file.directory:
+    - name: {{ tomcat.catalina_tmpdir }}
+    - user: {{ tomcat.user }}
+    - group: {{ tomcat.group }}
+    - makedirs: True
```

Comment:
In the extra testing I've done on top of this PR, I found that it's better that we set the `mode` here as well, otherwise it differs per platform.  So just the one line added in between.

```suggestion
    - group: {{ tomcat.group }}
    - mode: '755'
    - makedirs: True
```